etc/profile-a-l/hledger-ui.profile

```diff
@@ -0,0 +1,11 @@
+# Firejail profile for hledger-ui
+# Description: terminal interface (TUI) for hledger
+# This file is overwritten after every install/update
+# Persistent local customizations
+include hledger-ui.local
+
+# Allow /bin/sh (blacklisted by disable-shell.inc)
+# This seems required for launching editor in hledger-ui
+include allow-bin-sh.inc
```

Comment:
```suggestion
# This seems to be required for launching the editor in hledger-ui.
# Allow /bin/sh (blacklisted by disable-shell.inc)
include allow-bin-sh.inc
```

The `Allow ...` comment should always immediately precede the `include allow`
entry (see profile.template).

This also applies to the other PRs.